server/channels/utils/fileutils/fileutils_test.go

```diff
@@ -27,7 +26,10 @@ func TestFindFile(t *testing.T) {
 		//         tmpDir5/
 		tmpDir1, err := os.MkdirTemp("", "")
 		require.NoError(t, err)
-		defer os.RemoveAll(tmpDir1)
+		defer func() {
+			err := os.RemoveAll(tmpDir1)
+			require.NoError(t, err)
+		}()
```

Comment:
No need for a new `err` variable. You can just use the existing one
```suggestion
		defer func() {
			err = os.RemoveAll(tmpDir1)
			require.NoError(t, err)
		}()
```